content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,142 @@
+   going to :menuselection:`Purchase app --> Configuration --> Vendor Pricelists`. Click
+   :guilabel:`New` on the top left. Under the vendor column, add the product information as it
+   pertains to the vendor. If the product is being requested from this vendor, this information will
+   be pulled onto the RFQ so it can be easily found in their system.
+
+Ordering products through RFQ
+=============================
+
+With products and prices configured, follow these steps to create and send RFQs to make purchases
+for the company.
+
+RFQ dashboard
+-------------
+
+To get started, navigate to :menuselection:`Purchase app --> Orders --> Requests for Quotation`.
+From the RFQ dashboard, an overview of the company’s RFQs and Purchase Orders is available. The top
+of the screen breaks down all RFQs in the company as well as individual ones (where the user is the
+buyer) with a summary of their status. The top right corner also provides a quick report of the
+company’s recent purchases by total value, lead times, and number of RFQs sent.
+
+Filter and view options (e.g., Reference Number, Date, Vendor, etc.) allow toggling between
+different views to quickly sort and find orders. Just like in Odoo’s other apps, Filters and Groups
+can be customized in the search bar at the top of the page.
+
+Create new RFQ
+--------------
+
+To create a new RFQ, click the :guilabel:`New` button on the top left corner of the screen. A PO
+number will automatically be assigned to this new RFQ. From here, assign the vendor and order
+deadline, which lets the vendor know when they should confirm the quotation so that Odoo can
+confirm it into a PO.
+
+Additional fields
+~~~~~~~~~~~~~~~~~
+
+   :guilabel:`Vendor reference` points to the sales and delivery order numbers sent by the vendor.
+   This will come in handy once products are received and the PO needs to be matched to the delivery
+   order.
+
+   Under :guilabel:`Agreement`, blanket orders are long-term purchase agreements on recurring orders
+   with set pricing. To view and configure blanket orders, head to
+   :menuselection:`Purchase app --> Orders --> Purchase agreements`.
+
+   :guilabel:`Currency` can be changed if purchasing products from a vendor in another country.
+   To change the main currency or enable additional currencies, head to
+   :menuselection:`Configuration --> Settings --> Currencies`.
+
+   :guilabel:`Deliver to` provides options for the order shipment. Select the receiving warehouse
+   address here, or select :guilabel:`Dropship` to indicate that this order is to be shipped
+   directly to the end customer. When :guilabel:`Dropship` is selected, the
+   :guilabel:`Dropship address` field is enabled. Contact names will populate here from the
+   Contacts app.
+
+.. tip::
+   Expected Arrival is automatically calculated based on the Order Deadline and Vendor Lead Time.
+   Click the checkbox for :guilabel:`Ask confirmation` to ask for signage at delivery.
```

Comment:
Content in these admonition blocks are supplementary information. Since the expected arrival field is very important, I'd take it out and have it join its friends in the paragraph above.